django/db/models/aggregates.py

```diff
@@ -65,7 +65,15 @@ def resolve_expression(
         c.filter = c.filter and c.filter.resolve_expression(
             query, allow_joins, reuse, summarize
         )
-        if not summarize:
+        if summarize:
+            # Summarized aggregates cannot refer to summarized aggregates.
+            for ref in c.get_refs():
+                if query.annotations[ref].is_summary:
+                    raise FieldError(
+                        "Cannot compute %s('%s'): '%s' is an aggregate"
+                        % (c.name, ref, ref)
+                    )
+        elif not self.is_summary:
```

Comment:
This is required due to `Lookup.as_sql` performing expression resolving at compilation time.